texar/modules/classifiers/conv_classifiers.py

```diff
@@ -241,3 +241,17 @@ def layer_names(self) -> List[str]:
         r"""A list of uniquified layer names.
         """
         return self._encoder.layer_names
+
+    @property
+    def output_size(self) -> int:
+        r"""Return the final dimension of the output :attr:`logit`,
+        if the final dimension cannot be computed, return 1.
```

Comment:
The doc is inconsistent with the actual implementation when "the final dimension cannot be computed"

Reply:
Also, for a class property (`@property`), the docstring usually does not say "Return xxx"